javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javassistmodel/JavassistAnnotationDeclaration.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2016 Federico Tomassetti
```

Comment:
Perhaps we should come up with a new Copyright notice, we keep stating that and then forget about it :)

We can change it after this PR anyway

Reply:
I agree, let's do this after the PR. It needs to be done for many classes anyway. :-) I will add an issue about it.